src/plugins/substep/substep.js

```diff
@@ -88,9 +88,19 @@
             for ( var i = 0; i < substeps.length; i++ ) {
                 substeps[ i ].classList.remove( "substep-active" );
             }
-            var el = substeps[ visible.length ];
-            el.classList.add( "substep-visible" );
-            el.classList.add( "substep-active" );
+
+            var el;
+            var curr;
+            for ( var j = visible.length; j < substeps.length; j++ ) {
+                if ( curr && curr !== substeps[ j ].dataset.substepOrder ) {
+                    break;
+                }
```

Comment:
Please add a comment above the for to explain what  the for loop does (together with the if). It doesn't have to be long but it's needed.